plugin/pkg/scheduler/algorithmprovider/affinity/affinity.go

```diff
@@ -37,9 +37,9 @@ func affinityPredicates() util.StringSet {
 		"PodFitsResources",
 		"NoDiskConflict",
 		// Ensures that all pods within the same service are hosted on minions within the same region as defined by the "region" label
-		factory.RegisterFitPredicate("ServiceAffinity", algorithm.NewServiceAffinityPredicate(factory.PodLister, factory.ServiceLister, factory.MinionLister, []string{"region"})),
+		factory.RegisterFitPredicate("RegionAffinity", algorithm.NewServiceAffinityPredicate(factory.PodLister, factory.ServiceLister, factory.MinionLister, []string{"region"})),
```

Comment:
@bgrant0607 Since you mentioned @danmcp I would just like to get your quick input on something. If it turns out to be a more involved question, I'll create a separate issue.

Dan had asked me why we used services instead of replication controllers for identifying peer pods for spreading. The argument is that services might not be needed and could be avoided in some cases, whereas replication controller will most likely always be there. Technically, you could have a set of individual pods placed behind a service or pods from multiple replication controllers placed behind a service - but what is the most convenient/practical approach?

Thoughts???